### ISSUE TITLE
Crash when ruby-gettext isn't installed

### ISSUE THREAD

**Issue by kirbyfan64** [bleskodev/rubyripper#609]
```
ruby configure --update-mo #update the locale files
ruby-gettext is not found. Translations are disabled!
Could not import gettext; skip language support...
configure:109:in `update_mo': uninitialized constant GetText::Tools (NameError)
	from configure:138:in `block in <main>'
	from configure:124:in `each'
	from configure:124:in `<main>'
Makefile:10: recipe for target 'all' failed
make: *** [all] Error 1
```

¯\_(ツ)_/¯

**Comment by bleskodev** [comment]
Hi, thanks for reporting. I'll check it out and see what I can do.

**Comment by Warepire** [comment]
I ran into this too, I think I fixed this with PR #612

**Comment by bleskodev** [comment]
This should be fixed with PR #612 (thanks to Warepire!). Closing
